datadog/resource_datadog_monitor.go

```diff
@@ -232,8 +234,12 @@ func buildMonitorStruct(d *schema.ResourceData) *datadog.Monitor {
 		Options: &o,
 	}
 
-	if attr, ok := d.GetOk("enable_logs_sample"); ok && m.GetType() == "log alert" {
-		o.SetEnableLogsSample(attr.(bool))
+	if m.GetType() == "log alert" {
+		if attr, ok := d.GetOk("enable_logs_sample"); ok {
```

Comment:
Same for this string

Reply:
Since this is not a value, but an argument name, I'd actually prefer to keep it this way. The other argument names are used the same amount of times without having constants associated.